scripts/ci/avail-ext-doc/list-template.md

```diff
@@ -5,12 +5,9 @@ author: haroldrandom
 ms.author: jianzen
 manager: yonzhan,yungezz
 ms.date: {{ date }}
-ms.topic: article
+ms.topic: language-reference
```

Comment:
Would it be more suitable to be `hub-page`? or `generated-reference` if it's updated automatically on each extension release?